src/App.tsx

```diff
@@ -0,0 +1,184 @@
+        </Toolbar>
+      </AppBar>
+      <Container maxWidth="xl">
+      <Box sx={{ display: 'flex', flexDirection: 'row', mx: 'auto', my: 4}}>
+          <FormControl variant="outlined" sx={{minWidth: 120}}>
+              <InputLabel id="select-example">
+                Example
+              </InputLabel>
+              <Select
+                value={example}
+                onChange={handleSelectChange}
+                label="Example"
+                labelId='select-example'
+              >
+                {examples.map((name) => (
+                  <MenuItem value={name} key={name}>{name}</MenuItem>
+                ))}
+              </Select>
+          </FormControl>
+          <FormControl variant="outlined" sx={{minWidth: 120, mx: 2}}>
+              <InputLabel id="select-outputs">
+                Outputs
+              </InputLabel>
+              <Select
+              value={numOfOutputs}
+              onChange={(e) => setNumOfOutputs(Number(e.target.value))}
+              label="Outputs"
+              labelId='select-outputs'
+            >
+              <MenuItem value={0} key={0}>0</MenuItem>
+              <MenuItem value={1} key={1}>1</MenuItem>
+              <MenuItem value={16} key={16}>16</MenuItem>
+            </Select>
+          </FormControl>
+          <Button
+            variant="contained"
+            color="primary"
+            onClick={() => {
+              init().then(() => {
+                try {
+                  let resp = program(code, inputs, numOfOutputs);
+                  console.log(resp);
+                  setOutput({
+                    init: true,
+                    success: true,
+                    text: resp.join(" "),
+                  });
+                } catch (error) {
+                  setOutput({
+                    init: true,
+                    success: false,
+                    text: "Error: Check the developer console for details.",
+                  });
+                }
+              });
+            }}
+            endIcon={<SendIcon />}
+          >
+            Execute
+          </Button>
+          {output.init ? (
+            <Alert
+              variant="outlined"
+              severity={output.success ? "success" : "error"}
+              sx={{mx: 2}}
+            >
+              <Typography color="inherit" component="div">
+                {output.text}
+              </Typography>
+            </Alert>
+          ) : (
+            <></>
+          )}
+        </Box>
+        <Box sx={{ my: 4 }}>
+        <Typography color="primary" variant="h6">Inputs</Typography>
+          <CodeMirror
+            value={inputs}
+            height="100%"
+            theme={oneDark}
+            onChange={setInputs}
+          />
+        </Box>
+        <Box sx={{ my: 4 }}>
+        <Typography color="primary" variant="h6">Miden Assembly Code</Typography>
+          <CodeMirror
+            value={code}
+            height="100%"
```

Comment:
As suggested by @tohrnii, let's set a max height here. I think 500px is ok. We'll revisit later when upgrading the design anyway

```
maxHeight="500px"
```

Reply:
good point, somehow that slipped